### DISCUSSION TITLE
Renovate keeps reopening issue

### DISCUSSION THREAD

**Discussion by theS1LV3R** [renovatebot/renovate#8380] [Request Help] [answered]
I've recently added renovate to my [repo](https://github.com/ctfnote/backend) (great bot btw), but it has now created an [issue](https://github.com/CTFNote/backend/issues/35) that i can't seem to get rid of. 

I pushed a [commit](https://github.com/CTFNote/backend/commit/e5a015e911183e2d0d35262b80353e6d7d50a749) to fix the issue, but this didn't seem to work. The config was copy-pasted from the [docs page](https://docs.renovatebot.com/presets-schedule/#schedulemonthly), but the bot keeps reopening. I've tried both closing and locking the issue, but it just opens it again. Help?

**Comment by rarkins** [comment, marked as answer]
Renovate pulls its config from the *default* branch, and does not read it from any of the `baseBranches` you define (but there is a feature request open for that). Therefore you need to update this line still: https://github.com/CTFNote/backend/blob/24b6c544b1ca13853b037a16c1218b4cb6038a5f/renovate.json#L4

**Reply by theS1LV3R** [reply]
Ah, alright. Thanks!
